cleverhans/model_eager.py

```diff
@@ -0,0 +1,32 @@
+from cleverhans.model import Model
+from distutils.version import LooseVersion
+import tensorflow as tf
+
+
+if LooseVersion(tf.__version__) < LooseVersion('1.8.0'):
+    error = ('For eager execution',
+             'use Tensorflow version greather than 1.8.0.')
+    print(error)
+    exit(1)
+
+
+class Model_Eager(Model, tf.contrib.eager.Network):
```

Comment:
tfe.Network is deprecated in tensorflow rc 1.9.0. We could inherit from tf.keras.Model instead.

Reply:
#430 , suggestions were in favour of not using the keras dependencies.